packages/itwinui-react/src/core/Menu/MenuItem.test.tsx

```diff
@@ -177,17 +177,22 @@ it('should render sublabel', () => {
 });
 
 it('should show sub menu on hover', () => {
+  jest.useFakeTimers();
```

Comment:
Why do the timers have to be manually handled?

Reply:
probably because everything in jest is fake and it doesn't know to wait for the delay before menu appears.